Infrastructure/EntityConfigurations/EstablishmentStatusConfiguration.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Domain;
+using Domain.Constants;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Infrastructure.EntityConfigurations;
+
+public class EstablishmentStatusConfiguration : IEntityTypeConfiguration<EstablishmentStatus>
+{
+    public void Configure(EntityTypeBuilder<EstablishmentStatus> builder)
+    {
+        builder.HasData(
```

Comment:
Missing the somewhat pointless index on the name property

Reply:
Added